components/modal/src/modal-title/modal-title.js

```diff
@@ -9,12 +9,13 @@ export const ModalTitle = ({ children, dataTest }) => (
 
         <style jsx>{`
             h1 {
+                order: 1;
+                align-self: flex-start;
                 color: ${colors.grey900};
                 font-size: 20px;
                 font-weight: 500;
-                line-height: 24px;
+                line-height: ${spacers.dp24};
```

Comment:
I think this is a misuse of the spacing values. Although line height and spacing can be closely connected, they do have their differences. Until we have a dedicated theme system for font, I think we should use static values:

> `Line-height` and `vertical-align` are simple CSS properties. So simple that most of us are convinced to fully understand how they work and how to use them. But it’s not. They really are complex, maybe the hardest ones, as **they have a major role in the creation of one of the less-known feature of CSS: inline formatting context**.
> https://iamvdo.me/en/blog/css-font-metrics-line-height-and-vertical-align

Reply:
Thanks. I'll change it back.